pkgs/by-name/pr/presenterm/package.nix

```diff
@@ -22,19 +22,26 @@ rustPlatform.buildRustPackage (finalAttrs: {
     libsixel
   ];
 
+  buildFeatures = [
+    "sixel"
+  ];
+
   useFetchCargoVendor = true;
   cargoHash = "sha256-u0wOWKAfzi1Fxmx6x2ckrIv/PKgtqKrDiDauD4/BY24=";
 
+  checkFeatures = [
+    "sixel"
+  ];
   checkFlags = [
     # failed to load .tmpEeeeaQ: No such file or directory (os error 2)
     "--skip=external_snippet"
   ];
 
+  doInstallCheck = true;
   nativeInstallCheckInputs = [
     versionCheckHook
   ];
   versionCheckProgramArg = "--version";
-  doInstallCheck = true;
 
```

Comment:
Was that intentional?

Reply:
yes, I usually move `doInstallCheck` so it is the first attribute. If this is an issue, I can move it back.

Reply:
Fair enough, I put it back. I should not have made that change here anyway.